src/swupd.h

```diff
@@ -398,6 +399,23 @@ typedef enum telem_prio_t {
 } telem_prio_t;
 extern void telemetry(telem_prio_t level, const char *class, const char *fmt, ...);
 
+/* fs_storage.c */
+
+/* bundle_result contains the information to print along with
```

Comment:
This comment could use an update since it isn't just for outputting search information now.

Reply:
Agreed, will update for V2.